featurebyte/api/aggregator/forward_aggregator.py

```diff
@@ -81,12 +83,17 @@ def forward_aggregate(
             input_nodes=[forward_aggregate_node],
         )
         # Build and return Target
+        agg_method = construct_agg_func(agg_func=cast(AggFunc, method))
+        output_var_type = self.get_output_var_type(agg_method, method, value_column)
```

Comment:
I thought the var type is already derived from graph / operation structure automatically. Is that true @gee-senbong?

Reply:
It is derived from the operation structure during feature saving (required to call `FeatureModel` constructor, https://github.com/featurebyte/featurebyte/blob/f024c4b2a894abcd103bc0337173f030d6bdb2e3/featurebyte/models/feature.py#L224-L252). Currently, it is not derived automatically for SDK API object.